src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/LostBedsTest.kt

```diff
@@ -29,7 +29,7 @@ class LostBedsTest : InitialiseDatabasePerClassTestBase() {
   @Nested
   inner class GetAllLostBeds {
     @Test
-    fun `Get All Lost Beds without JWT returns 401`() {
+    fun `List Lost Beds without JWT returns 401`() {
```

Comment:
Should these tweaks to the spec titles actually be part of the previous PR 1900 https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1909/files#diff-0a762b45881962842eff1ea864b80ee2b8c8b9923d96a176061d01c36d48b9ab

Reply:
They were an artifact of rebasing the branch for #1909 without updating this branch as well